### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NewNotebook_ScreenReader:Voice Over is  announcing the button for "Kernal" combo box.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5679]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
Environment Details:
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS : MacOS Mojave 10.14.5

Additional Details:                                                                                                                                                                                                                                                                                                                                                                                                        
Tools Name: VoiceOver
MAS Number :MAS1.3.1,MAS4.1.2

**Steps to Reproduce :**
1.Install the app Azure Data Studio.
2.Connect to the Database->Go to file->New Notebook.
3.Start Voiceover.
4.Navigate to the "Kernal " Combo box.
5.Observe that Screen reader is announcing button for "kernal" combo box.

**Actual:**
Navigate till "Kernal" Combobox, screen reader announce button for "Kernal Combobox".

**Same issue observed with combo box for “Attach To”.**

**Expected:**
Screen Reader should announce  Combobox for "Kernal "combobox .Same for  “Attach To” combobox.

**User Impact:**
Screen reader only users find it difficult to understand what is purpose of combobox.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx


**MAS Reference:**
**MAS1.3.1-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**MAS4.1.2 -**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={248054a6-5e68-4771-9e1e-242fb5025730}

Attachment for Reference:
[5679_[Accessibility]A11y_AzureDataStudio_NewNotebook_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3226724/5679_.Accessibility.A11y_AzureDataStudio_NewNotebook_ScreenReader.pptx)


Does this issue occur when all extensions are disabled?: Yes

**Comment by AccessibilityTestingTeam-TCS** [comment]
Closing this bug as this bug is the regular behaviour of MAC as per bug linked.
This is also confirmed behaviour in VS code application. Hence closing the bug.

**Comment by AccessibilityTestingTeam-TCS** [comment]
I checked this issue on following environment
Version: 1.10.0-insider (user setup)
Commit: f720ec642f6ffd51d8772f663a611292f91a563d
Date: 2019-07-15T07:47:15.691Z
VS Code 1.34.0
Electron: 3.1.8
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS : MacOS Mojave 10.14.5
Closing this bug as this bug is the regular behaviour of MAC as per bug linked.
This is also confirmed behaviour in VS code application. Hence closing the bug.
